src/client_side_request.cc

```diff
@@ -1134,6 +1134,16 @@ clientInterpretRequestHeaders(ClientHttpRequest * http)
         s.clean();
     }
 
+    // headers only relevant to reverse-proxy
+    if (request->flags.accelerated) {
+        // check for surrogate_id value in the CDN-Loop header (if any)
+        if (req_hdr->hasListMember(Http::HdrType::CDN_LOOP, Config.Accel.surrogate_id, ',')) {
+            debugObj(33, DBG_IMPORTANT, "WARNING: Forwarding loop detected for:\n",
```

Comment:
Done.

The net_db request does not pass through the client_side code, it goes directly to FwdState fetching. So I see no reason why it would be affected by these changes.